core/src/main/java/com/microsoft/applicationinsights/telemetry/ExceptionTelemetryOptions.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.applicationinsights.telemetry;
+
+public class ExceptionTelemetryOptions {
+    private int maxStackSize;
+    private int maxTraceLength;
+
+    public ExceptionTelemetryOptions(Integer maxStackSize, Integer maxTraceLength) {
```

Comment:
I think it's good to comment out that default values would be Integer Max for future readers.

Reply:
I've added the description of default values into class level javadoc.